### ISSUE TITLE
Shortwave can not launched from application menu !

### ISSUE THREAD

**Issue by Nokia808** [flathub/de.haeckerfelix.Shortwave#2]
Hi. 1st of all I would like to thank you for making this application available as flatpak on FlatHub.

I'm using Fedora Linux version 30 X64 bit Cinnamon edition. I installed Shortwave but I can not launch it from application menu ! When I clicked on it's icon in my application menu, no thing happened ! I tried so many times, I restart my PC & retried again but no thing happened !

I'm only able to launch Shortwave from terminal using the following command:
`flatpak run de.haeckerfelix.Shortwave`

Please your kind fix.

**Comment by Nokia808** [comment]
Hi. Now, I can launch it without problem from it's icon in my Cinnamon applications menu !! How this happened ?!! I did not do any think from my side, & I did not receive an update from FlatHub !! 

I do not know what had been happened ! And, thus, I do not know whether I have to close this issue or not. Yesterday, there was error messages appear in terminal when I launch program using command line. Unfortunately, I did not copy them, but I remember that they contain "GTK" in yellow colour ....
